docs/documentation/github-desktop.md

```diff
@@ -0,0 +1,38 @@
+# Store your code online with GitHub and GitHub Desktop
+
+GitHub is a way to store code online so you can collaborate with other people. It also makes it easier to publish your prototype online with Heroku.
+
+Some concepts:
+
+- Git - a way to store versions of your code
+- GitHub - a website that shares your code online
+- GitHub Desktop - an app that puts your code on GitHub
+- Repository/repo - a project that has been stored in Git
+
+## Set up GitHub Desktop
+
+1. Create a GitHub account, it's free: https://github.com/join
```

Comment:
Could we say "Create a free GitHub account"? And link the whole sentence to https://github.com/join?